mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergOutputCommitter.java

```diff
@@ -201,22 +203,22 @@ public void writerIsClosedAfterTaskCommitFailure() throws IOException {
         .when(failingCommitter)
         .commitTask(argumentCaptor.capture());
 
-    Table table = table(temp.getRoot().getPath(), false);
+    Table table = table(temp.toFile().getPath(), false);
     JobConf conf = jobConf(table, 1);
 
     Assertions.assertThatThrownBy(
             () -> writeRecords(table.name(), 1, 0, true, false, conf, failingCommitter))
         .isInstanceOf(RuntimeException.class)
         .hasMessage(exceptionMessage);
 
-    Assert.assertEquals(1, argumentCaptor.getAllValues().size());
+    Assertions.assertThat(argumentCaptor.getAllValues().size()).isEqualTo(1);
```

Comment:
same as previously mentioned about size assertions: `hasSize(xyz)` is better here